component/otelcol/exporter/prometheus/internal/convert/convert_test.go

```diff
@@ -318,9 +318,10 @@ func TestConverter(t *testing.T) {
 		},
 	}
 
+	decoder := &pmetric.JSONUnmarshaler{}
```

Comment:
Does this keep any state? Is it safe to share across all tests?

Reply:
No, the `JSONUnmarshaler` is an empty struct, at least for now. We recreate it just in case this changes in the future, but for now it's fine.